### ISSUE TITLE
[Feature] Add the ability to switch to the old translation panel style on the phone.

### ISSUE THREAD

**Issue by Stikulzon** [FilipePS/Traduzir-paginas-web#801]
### Is your feature request related to a problem? Please describe

Not applicable.

### Describe the solution you'd like

Add the ability to switch to the old translation panel style on the phone. 

### Additional context

The old translation panel style was much more suitable for me because of its small size and the appearance of a solid line instead of a bubble.

### Did you fill in all the fields correctly?

- [X] Yes, everything is fine!

**Comment by FilipePS** [comment]
With the current notification style interface, after clicking translate you can quickly close it by dragging it to the side. Do you feel the need to keep the popup always on the screen?

**Comment by Stikulzon** [comment]
Yes. Due to the imperfection of the translation, I very often switch to the original and back

**Comment by KishanPRao** [comment]
I prefer the original layout too, as it didn't noticeably block a larger part of the page (bottom areas were rarely important for me).
And, switching back and forth was very simple & useful.
